features/faq.mdx

```diff
@@ -42,3 +42,34 @@ Yes, we do support on-premises deployment for a select few of our enterprise cus
 
 If you are interested in onprem, please [contact us](mailto:hello@promptlayer.com) for more information.
 
+## Does AsyncOpenAI work with PromptLayer?
+
+Yes, AsyncOpenAI works with PromptLayer. You can use AsyncOpenAI with PromptLayer as shown below.
```

Comment:
Yes, AsyncOpenAI is compatible with PromptLayer. Use them together as demonstrated in example below.

Reply:
@jzone3updated.